BedrockServer.cpp

```diff
@@ -33,7 +33,16 @@ void BedrockServer::cancelCommand(const string& commandID) {
 }
 
 bool BedrockServer::canStandDown() {
-    return _writableCommandsInProgress.load() == 0;
+    
+    size_t httpsCommands = 0;
+    {
+        lock_guard<mutex> lock(_httpsCommandMutex);
+        httpsCommands = _outstandingHTTPSRequests.size();
+    }
+    if (httpsCommands) {
```

Comment:
I'm worried about this. Some HTTPS requests we have last up to 5 minutes, and I don't think we want stand down hanging for this long. Can we add a time out here? AFAICT the worst thing that would happen is the https requests returns after we've stood down, and then we'll warn and do nothing with the results.

Reply:
Although https://github.com/Expensify/Bedrock/blob/master/sqlitecluster/SQLiteNode.cpp#L931-L936 may actually be the correct place to add the timeout.